LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Mac/LinkADRCmd.cs

```diff
@@ -13,6 +13,11 @@ public class LinkADRCmd : GenericMACCommand
         // private readonly byte dataRateTXPower;
         private readonly byte[] chMask = new byte[2];
 
+        public LinkADRCmd()
```

Comment:
Just to understand: the virtual Length will come for next release right?

Reply:
Yes